deps.edn

```diff
@@ -892,7 +895,18 @@
   :db/mariadb-latest
   {:jvm-opts
    ["-Dmb.db.type=mysql"
-    "-Dmb.db.connection.uri=jdbc:mysql://localhost:3307/metabase_test?user=root&password="]}}}
+    "-Dmb.db.connection.uri=jdbc:mysql://localhost:3307/metabase_test?user=root&password="]}}
+
+  ;; ./bin/mage start-db mongo oldest
+  :db/mongo-oldest
+  {:jvm-opts
+   ["-Dmb.db.type=mongo"
+    "-Dmb.db.connection.uri=jdbc:mongo://localhost:27017"]}
+  ;; ./bin/mage start-db mongo latest
+  :db/mongo-latest
+  {:jvm-opts
+   ["-Dmb.db.type=mongo"
+    "-Dmb.db.connection.uri=jdbc:mongo://localhost:27018"]}}
```

Comment:
as discussed on Slack, you're right that we should drop these and print the env vars that will do this, basically: https://github.com/metabase/dev-scripts/blob/master/run-mongo.sh#L18-L24